data/json/itemgroups/clothing.json

```diff
@@ -188,5 +188,90 @@
       [ "pocketwatch", 25 ],
       [ "wristwatch", 100 ]
     ]
+  },
+  {
+    "id": "clothing_technician",
+    "type": "item_group",
+    "subtype": "distribution",
+    "items": [
+      { "collection": [
+        { "item": "technician_shirt_blue" },
+        { "item": "technician_pants_blue" }
+      ], "prob": 50 },
+      { "collection": [
+        { "item": "technician_shirt_ltblue" },
+        { "item": "technician_pants_ltblue" }
+      ], "prob": 30 },
+      { "collection": [
+        { "item": "technician_shirt_gray" },
+        { "item": "technician_pants_gray" }
+      ], "prob": 20 }
+    ]
+  },
+  {
+    "id": "clothing_work_boots",
+    "type": "item_group",
+    "subtype": "distribution",
+    "entries": [
+      { "item": "boots", "prob": 70 },
+      { "item": "boots_hiking", "prob": 10 },
+      { "item": "boots_steel", "prob": 20 }
+    ]
+  },
+  {
+    "id": "clothing_work_glasses",
+    "type": "item_group",
+    "subtype": "distribution",
+    "entries": [
+      { "item": "glasses_safety", "prob": 85 },
```

Comment:
Sounds reasonable. We need to ensure there's a fairly constant supply of (damaged) glasses for players with short and long-sighted traits.